omnigibson/prims/cloth_prim.py

```diff
@@ -88,8 +108,31 @@ def _post_load(self):
         if "mass" in self._load_config and self._load_config["mass"] is not None:
             self.mass = self._load_config["mass"]
 
+        # Save the default point configuration
+        self.save_configuration("default", self.points)
+
+        # Remesh the object if necessary
+        force_remesh = self._load_config.get("force_remesh", False)
```

Comment:
what's "force_remesh" for?

Reply:
In the pipeline, we need to force this class to remesh the object even though the scale is 1, 1, 1. That also happens through this logic so we need to allow forcing it.